src/controllers/sessionController.ts

```diff
@@ -97,7 +116,7 @@ export default class SessionController {
             myPortalToken: updatedUser.myportal_token,
             spidLevel: updatedUser.spid_level,
             walletToken: updatedUser.wallet_token,
-            zendeskToken: updatedUser.zendesk_token
+            zendeskToken: updatedUser.zendesk_token + zendeskSuffix
```

Comment:
ditto

Reply:
fixed in https://github.com/pagopa/io-backend/pull/926/commits/ca533a71f9f89340555f1e21f73156c352de4b6c